src/Hamiltonians/excitations.jl

```diff
@@ -83,6 +83,60 @@ function momentum_transfer_excitation(add::FermiFS, chosen::Integer, map; fold=t
     return add, 0.0, 0, 0, 0
 end
 
+function extended_momentum_transfer_excitation(add::SingleComponentFockAddress, chosen::Integer, map; fold=true)
```

Comment:
This is the same function as `momentum_transfer_excitation`. Delete this one and use `momentum_transfer_excitation` instead.

Reply:
That's true, but the problem is that the function ``momentum_transfer_excitation(address:: FermiFS)``is also defined, which returns ``new_addr, 0.0, 0.0, 0.0, 0.0``, which we do not want. That is the reason why I defined the same thing with a different name.

Reply:
Please let me know if there is a more efficient way.